packages/fxa-settings/src/components/Settings/UnitRowTwoStepAuth/index.tsx

```diff
@@ -209,7 +212,14 @@ export const UnitRowTwoStepAuth = () => {
           />
         )}
         {disable2FAModalRevealed && (
-          <DisableTwoStepAuthModal {...{ hideDisable2FAModal }} />
+          <MfaGuard
```

Comment:
@nshirley This seems to be all that's needed to wrap a button. Note that in this scenario the `onDismissCallback` is needed. This was recently added in https://github.com/mozilla/fxa/pull/19443.